ignite/cmd/node_query_bank_balances.go

```diff
@@ -0,0 +1,61 @@
+package ignitecmd
+
+import (
+	"fmt"
+
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/spf13/cobra"
+)
+
+func NewNodeQueryBankBalances() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "balances [account-name|address]",
```

Comment:
I think this should be a similar format as in the `tx send` cmd:

```
[from_account_or_address]
```

just for consistency

Reply:
done [a9b9f9b](https://github.com/ignite/cli/pull/2669/commits/a9b9f9b9493b9ee0b2e4e37e50d05ae43b877c4b)